packages/salesforcedx-sobjects-faux-generator/src/transformer/transformerFactory.ts

```diff
@@ -93,7 +93,7 @@ export class SObjectTransformerFactory {
   ): Promise<Connection> {
     const connection = await Connection.create({
       authInfo: await AuthInfo.create({
-        username: await ConfigUtil.getUsername(projectPath)
+        username: (await ConfigUtil.getUsername(projectPath)) as string
```

Comment:
If we change the return of getUsername to Promise<string | undefined>, then this change can be removed